### ISSUE TITLE
make clean localnet-start error's if no existing data dir

### ISSUE THREAD

**Issue by arnabmitra** [provenance-io/provenance#435]
## Summary of Bug

make clean localnet-start error's if no existing data dir

## Version

v1.5.0

## Steps to Reproduce

make clean localnet-start
rm -rf /Users/arnabmitra/provenance/build/*
rm: /Users/arnabmitra/provenance/build/node0/data: Directory not empty
rm: /Users/arnabmitra/provenance/build/node0: Directory not empty
rm: /Users/arnabmitra/provenance/build/node2/data: Directory not empty
rm: /Users/arnabmitra/provenance/build/node2: Directory not empty
make: *** [clean] Error 1

should just ignore the error codes if nothing exists in data imo.

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by dwedul-figure** [comment]
I played around with this and can't reproduce it. I tried on `main`, `v1.7.6`, and `v.1.5.0`. I tried starting it (`make clean localnet-start`), stopping it (`make localnet-stop`), then starting twice in a row (`make clean localnet-start` x 2), and finally starting it again without the clean (`make localnet-start`).

The output above makes me think you've got something special set up for `rm` that makes it do different things. Specifically, `rm -rf <dir>` should delete the directory as well as its contents, even if it's got stuff in it. But it's complaining that sub-directories aren't empty (which `-f` should make it ignore).
